File: sample/src/main/java/ru/terrakok/cicerone/sample/mvp/bottom/forward/ForwardView.java
package ru.terrakok.cicerone.sample.mvp.bottom.forward;


import moxy.MvpView;
import moxy.viewstate.strategy.AddToEndSingleStrategy;
import moxy.viewstate.strategy.StateStrategyType;

/**
 * Created by terrakok 26.11.16
 */

@StateStrategyType(AddToEndSingleStrategy.class)
public interface ForwardView extends MvpView {
    void setChainText(String chainText);
}

File: sample/src/main/java/com/github/terrakok/cicerone/sample/mvp/bottom/forward/ForwardView.kt
package com.github.terrakok.cicerone.sample.mvp.bottom.forward

import moxy.MvpView
import moxy.viewstate.strategy.AddToEndSingleStrategy
import moxy.viewstate.strategy.StateStrategyType

/**
 * Created by terrakok 26.11.16
 */
@StateStrategyType(AddToEndSingleStrategy::class)
interface ForwardView : MvpView {
    fun setChainText(chainText: String)
}